plugins/modules/identity/keycloak/keycloak_user_info.py

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2022
```

Comment:
You need to add someone who owns that copyright, like yourself. If you copied most of the module's code from another code, also include copyright lines from these modules.